autoscale/src/test/java/com/sequenceiq/periscope/modul/rejected/RejectedThreadContext.java

```diff
@@ -59,6 +60,7 @@
     @MockBean({Clock.class, ClusterService.class, CloudbreakClientConfiguration.class,
             MetricUtils.class, InternalCrnBuilder.class, FailedNodeRepository.class})
     @EnableAsync
+    @Profile("devtest")
```

Comment:
May have asked this earlier; is this annotation required?

Reply:
Nested SpringConfig in RejectedThreadContext is basically a TestContext.  Without the profile filter this would result in duplicate  bean definition for periscopeListeningScheduledExecutorService and fails integration endpoint tests which run tests against embedded tomcat and embedded db. Earlier there wasn't any service integration tests hence the missing annotation did not cause any issue.